src/domains/idpay-common/99_variables.tf

```diff
@@ -240,6 +240,29 @@ EOD
   }))
 }
 
+#Redis
+
+variable "redis_capacity" {
```

Comment:
@dariopelliccioli In my opinion, it's better to have a configuration object containing the useful variables as properties.. but it's just a matter of preference ;-)